api/routes/ACTION_router.js

```diff
@@ -0,0 +1,84 @@
+const router = require('express').Router();
+
+// MODELS
+const actionModel = require('../../data/helpers/actionModel.js')
+
+// - GET - // -- get() requires an id 
+router.get('/:id', (req, res) => {
+    console.log('actionRouter get/')
+    const {id} = req.params
+
+    actionModel.get(id)
+    .then( getResult => {
+        res.status(200).json( getResult )
+    })
+    .catch( err => {
+        console.log(err)
+        res.status(500).json(err)
+    })
+})
+// - POST - // Actions Schema are notes and description
+router.post('/:project_id', (req, res) => {
+    console.log('actionRouter post/')
+    const {project_id} = req.params
+
+    const objToPass = {
+        notes: req.body.notes,
+        description: req.body.description,
+        project_id
+        
+    }
+    console.log(objToPass)
+
+    if (!objToPass.notes && !objToPass.description && !objToPass.description.length < 128) {
```

Comment:
this 'validation' would be a great example of a middleware (lines 23 - 31 would be elsewhere) / helper function (you would essentially import it and use it use it here. easier to read (modular) and debug